tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -620,15 +629,32 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         }
       },
       observers: [
-        ('_updateSelectedDashboardFromActive(' +
-         '_selectedDashboard, _activeDashboards)'),
+        ('_setDefaultSelectedDashboardFromActive(' +
+         '_selectedDashboard, _selectedDashboardInitialized, _activeDashboards)'),
+        ('_persistSelectedDashboard(' +
+         '_selectedDashboard, _selectedDashboardInitialized)'),
         ('_ensureSelectedDashboardStamped(' +
          '_dashboardRegistry, _dashboardContainersStamped, ' +
          '_activeDashboards, _selectedDashboard)'),
         ('_updateDataSelection(_dashboardRegistry, _dataSelection,' +
          '_activeDashboards, _selectedDashboard)'),
       ],
 
+      async attached() {
+        const tabValue = await tf_storage.getString(tf_storage.TAB);
+        this._selectedDashboard = tabValue;
+        this._selectedDashboardInitialized = true;
+      },
+
+      _persistSelectedDashboard() {
```

Comment:
Why are these analytics changes in this commit?

Reply:
I forgot to remove `_selectedDashboardChanged`. Thanks for catching that. 

As to the behavioral change, my rationale is that analytics have to roughly match that of the URL change. Since we now persist `selectedDashboard` prop in the storage after first _real_ value is set, I thought it makes sense to move the `ga` here too. 

Technically, this is a behavior change, as you have indicated, since we would no longer have "/" pageviews if user loaded TensorBoard with a hash in the URL. I am willing to revert the change back but I am afraid of double logging of "/".

Reply:
Edit: reverted back to the older behavior.